asciidoctor-inline-parser.gemspec

```diff
@@ -1,7 +1,7 @@
 require File.expand_path('lib/asciidoctor/inline_parser/version', __dir__)
 
 Gem::Specification.new do |s|
-  s.name     = 'asciidoctor-templates-compiler'
+  s.name     = 'asciidoctor-inline-parser'
```

Comment:
Ah ah ah I took `asciidoctor-templates-compiler` as a starter project to initialize the project 🤣